src/storage/new_txn/new_txn.cpp

```diff
@@ -2645,6 +2746,75 @@ Status NewTxn::PostRollback(TxnTimeStamp abort_ts) {
         }
     }
 
+    const Vector<UniquePtr<MetaKey>> &metas = txn_store_.GetMetaKeyForBufferObject();
```

Comment:
1. Add comments which cases will need to clean up?
2. This part should be refactor in future, according to txn store.